automation_py/automations/gmail_creator/terms_handler.py

```
@@ -580,24 +583,91 @@ def _handle_checkbox_terms(self) -> bool:
+            for modal_button in modal_indicators:
+                try:
+                    if self._element_exists(modal_button, timeout=2):
+                        logger.info(
+                            f"[OK] Segundo modal detectado! Clicando em: {modal_button}")
+                        self.driver.find_element(
+                            By.XPATH, modal_button).click()
+                        logger.info(
+                            "[OK] Segundo botão de confirmação clicado")
+                        time.sleep(2)
+                        break
+                except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)